### DISCUSSION TITLE
Add Ardy cloak & Explorer's ring teleport charges to overlay.

### DISCUSSION THREAD

**Discussion by AndresConforti** [runelite/runelite#14981] [Ideas]
Hey there, i have seen that there is no integrated plugin nor in the hub that does that.
Shouldn't it be already an option for that in the "Integrated Charges" plugin?

I mean both those items are used for farm runs on most cases (On accounts without the hard diaries completed)

**Comment by Nightfirecat** [comment]
We'd like to add this to the client, #14723 already exists for tracking this.
